microsoft-edge/devtools-guide-chromium/css/inspect.md

```diff
@@ -0,0 +1,82 @@
+   An element displayed using CSS grid layout showing an extra icon next to its name in the Inspect overlay
+:::image-end:::
+
+:::image type="complex" source="images/inspect-tool-flexbox-element.msft.png" alt-text="An element displayed using CSS flexbox showing an extra icon next to its name in the Inspect overlay" lightbox="images/inspect-tool-flexbox-element.msft.png":::
+   An element displayed using CSS flexbox showing an extra icon next to its name in the Inspect overlay
+:::image-end:::
+
+The **Accessibility** section of the **Inspect** overlay displays information about text-color contrast, the name and the role of the element reported to assistive technoloy and whether the element is keyboard focusable or not. 
+
+For example, in the demo page, the `Bad Contrast` button shows a warning icon next to the contrast value of 1.77. It also shows that the button is not focusable via keyboard as it is a `DIV` element with a class of button instead of a `BUTTON` element.
+
+:::image type="complex" source="images/inspect-tool-bad-contrast.msft.png" alt-text="Elements without sufficient contrast show a warning icon" lightbox="images/inspect-tool-bad-contrast.msft.png":::
+   Elements without sufficient contrast show a warning icon
+:::image-end:::
+
+## Inspecting non-accessible elements
+
+Elements that are covered by other elements or have the CSS property of `pointer-events: hidden` are not available to the **Inspect** tool. In the demo page, try to hover over the `Overlay Button` and you will see that the parent element - in this case `div.wrapper` is shown instead.
+
+:::image type="complex" source="images/inspect-tool-element-element-without-pointer-events.msft.png" alt-text="Trying to select an element with pointer `events: none` is not possible." lightbox="images/inspect-tool-element-element-without-pointer-events.msft.png":::
+   Trying to select an element with `pointer events: none` is not possible.
+:::image-end:::
+
+If you want to inspect elements like that, you need to press `Shift` whilst hovering over elements. There is also a color overlay showing you that you are in an advanced selection mode.
+
+:::image type="complex" source="images/inspect-tool-with-shift.msft.png" alt-text="Pressing the Shift key whilst selecting elements on the page allows you to access elements more elements, even those with a pointer-events: none style setting" lightbox="images/inspect-tool-with-shift.msft.png":::
+   Pressing the Shift key whilst selecting elements on the page allows you to access elements more elements, even those with a pointer-events: none style setting
+:::image-end:::
+
+## Selecting the element and terminating Inspect mode
+
+If you click on any of the elements, the **Styles** tool shows the CSS applied to the element and the **Inspect** tool deactivates.
```

Comment:
Maybe also mention that the element becomes selected in the DOM tree?

Reply:
fixed